streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedStream.java

```diff
@@ -1509,7 +1509,7 @@
      * @return a windowed {@link KTable} that contains "update" records with unmodified keys, and values that represent
      * the latest (rolling) aggregate for each key within a window
      * @deprecated use {@link #windowedBy(SessionWindows) windowedBy(sessionWindows)} followed by
-     * {@link SessionWindowedKStream#aggregate(Initializer, Aggregator, Merger, Materialized) aggregate(initializer, aggregator, sessionMerger, Materialized.as("someStoreName").withValueSerde(aggValueSerde))}
+     * {@link SessionWindowedKStream#aggregate(Initializer, Aggregator, Merger, Materialized) aggregate(initializer, aggregator, sessionMerger, Materialized.with(keySerde, aggValueSerde))}
```

Comment:
This should be `Materialized.with(null, aggValueSerde)` -- there is no `keySerde` parameter in the deprecated method. The idea is to show a 1:1 translation to the new API.

Reply:
And I forgot, there are three `aggregate` methods in this class that need this update.

Reply:
Yep, in that case we also need to set the `keySerde` on the `Materialized` to the one provided in `KGroupedStream` etc. Otherwise it is going to revert to the defaults, which may not be correct

Reply:
Yes. Using `null` implies fall back to default -- that is the same behavior as the deprecated method has (the deprecate method does not take a `keySerde` argument and thus used the default one)